Scripts/build-derived-assets.sh

```
@@ -0,0 +1,20 @@
+#!/bin/sh
```

I always suggest "-eu" on sh/bash shebangs. The "-e" means fail immediately if any sub-command fails and "-u" means fail if any undefined variables are used (rather than assuming "" if undefined). Has saved me many times in the past from thinking things were working when they actually were not.

---

This script is not executed from a command line, so the shebang args will not apply. I can add them to the shell command in the project file, though.

---

Nevermind. The way we're referencing the script means it does get called as a shell command; I'll add those flags to ours scripts.

---

:+1: